api/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for api project.
+
+Generated by 'django-admin startproject' using Django 2.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '6q(#sk))z_y(jpc9pi4u4%$5)p)g_j_b2*7oaha@wy7g3wmtc5'
```

Comment:
not so secret. usar variavel de ambiente ou algo do genero.

Reply:
Vdd, vou colocar no envdir